eng/scripts/CodeChecks.ps1

```diff
@@ -94,6 +96,45 @@ try {
         & $PSScriptRoot\Export-API.ps1 -ServiceDirectory $ServiceDirectory -SDKType $SDKType -SpellCheckPublicApiSurface:$SpellCheckPublicApiSurface
     }
 
+    Write-Host "Validating installation instructions"
+    Join-Path "$PSScriptRoot/../../sdk" $ServiceDirectory  `
```

Comment:
`Join-Path` is pretty dumb. I'd use `[System.IO.Path]::Combine()`. If `$ServiceDirectory` is an absolute folder (can't remember if we supported that in these scripts, but we do in some places in the repo) then it'll be used as-is instead. `Join-Path` would just concat them for an invalid path.

Reply:
serviceDirectory is just the name of the service folder, e.g. "eventgrid"

Reply:
Some scripts - like the TestResources script - do also accept a full directory spec. These may not, but in general I stay away from `Join-Path`. I opened a bug years ago and the PowerShell team agreed that for the FileSystem provider they probably should've used `Path.Combine` but that it'd be a breaking change.

Reply:
To note, though, I'm not saying this has to be fixed. More of "tip".

Reply:
Yeah, I actually just copied this from another place in the same script where we are using Join-Path. But good to know about the potential pitfalls.